### PR TITLE
In DEBUG mode print the results of the project search

### PR DESCRIPTION
**Pull request by xrmx**
To match what we are doing in search/

### CODE REVIEW

File: readthedocs/projects/views/public.py [hunk 1]
```diff
@@ -332,6 +333,9 @@ def elastic_project_search(request, project_slug):
     else:
         results = {}
 
+    if settings.DEBUG:
+        print(pprint(results))
```
Comment (stsewd) [inline readthedocs/projects/views/public.py]: Isn't better to use log.debug?

**Comment by xrmx** [comment]
To me it was to debug the code that parses the elasticsearch response.

**Comment by humitos** [comment]
Got it. So, since it's a code that it's good for debugging, I think we shouldn't add this to the repo since it will be running in production and it's not necessary there. I think we can close this PR.

Thanks!

**Comment by xrmx** [comment]
@humitos Thanks for answering but:
- it's under settings.DEBUG
- the very same code is already there in search/ for other code paths querying elasticsearch
- the code was actually useful when debugging RTD code

File: readthedocs/projects/views/public.py (hunk 1 above)
Comment (xrmx) [reply readthedocs/projects/views/public.py]: Agree It may be a better option

**Comment by xrmx** [comment]
So given this is just useful for debugging, your review bandwidth limited and a cause for conflicts on more interesting PRs I'm closing this.

**Comment by RichardLitt** [comment]
Thanks, @xrmx, for both the PR and for being considerate and outlining why you're closing this.

**Pull request closed without merging**